test/byContract/registries/registries.test.ts

```diff
@@ -316,28 +318,22 @@ describe('Registries', () => {
             );
             await expect(appRegistry.createApp(appProvider.address, ...createAppArgs))
                 .to.emit(appRegistry, 'Transfer')
-                .withArgs(
-                    AddressZero,
-                    appProvider.address,
-                    ethers.BigNumber.from(predictedAddress).toString(),
-                );
+                .withArgs(ZeroAddress, appProvider.address, BigInt(predictedAddress).toString());
```

Comment:
Hardhat automatically converts it to `bigint`
```suggestion
                .withArgs(ZeroAddress, appProvider.address, predictedAddress);
```